HousingSearchListener.Tests/V1/E2ETests/Steps/UpdateProcessSteps.cs

```diff
@@ -2,6 +2,7 @@
 using FluentAssertions;
 using Hackney.Shared.HousingSearch.Domain.Process;
 using Hackney.Shared.HousingSearch.Gateways.Models.Processes;
+using Hackney.Shared.Processes.Sns;
```

Comment:
You need to add a line to `ThenTheIndexIsUpdatedWithTheProcess` to check that the StateStartedAt value is populated

Reply:
resolved by the suggestion given below:
```
process.StateStartedAt.Should().NotBeNull();
process.StateStartedAt.Should().Be(_stateChangeData.StateStartedAt.ToString());
```